wdl/transforms/cascades/src/main/scala/wdl/transforms/cascades/wdlom2wom/package.scala

```diff
@@ -15,13 +29,118 @@ import wdl.transforms.cascades.linking.expression.consumed._
 import wdl.transforms.cascades.linking.expression.files._
 import wdl.transforms.cascades.linking.expression.types._
 import wdl.transforms.cascades.linking.expression.values._
+import wom.values.WomInteger
+import wom.{CommandPart, RuntimeAttributes, RuntimeAttributesKeys}
 
 package object wdlom2wom {
   val taskDefinitionElementToWomTaskDefinition: CheckedAtoB[TaskDefinitionElementToWomInputs, CallableTaskDefinition] =
-    CheckedAtoB.fromErrorOr(TaskDefinitionElementToWomTaskDefinition.convert)
+    CheckedAtoB.fromErrorOr(convert)
   val workflowDefinitionElementToWomWorkflowDefinition
     : CheckedAtoB[WorkflowDefinitionConvertInputs, WorkflowDefinition] =
     CheckedAtoB.fromErrorOr(WorkflowDefinitionElementToWomWorkflowDefinition.convert)
   val fileElementToWomBundle: CheckedAtoB[FileElementToWomBundleInputs, WomBundle] =
     CheckedAtoB.fromCheck(FileElementToWomBundle.convert)
+
+  private def convert(b: TaskDefinitionElementToWomInputs)(implicit
```

Comment:
Only about lines 70 to 101 actually deal specifically with return code stuff – would it be possible to split up this function a bit more and keep more of the generalized stuff in its original home in `wdl.transforms.base.wdlom2wom.TaskDefinitionElementToWomTaskDefinition#convert`?